views/order/view.hbs

```diff
@@ -0,0 +1,74 @@
+<h2>Order {{order.id}}</h2>
+
+<div class="row mb-3">
+	<div class="col">
+		<div class="card card-body">
+			{{#with order}}
+			<h3>Order Details</h3>
+
+			<p>Date: {{datetime Date}}</p>
+			<p>Status: {{uncamel Status}}</p>
+			<p>Total cost: {{{price Total}}}</p>
+<!--
+			<p>Indivdual: User</p>
+			<p>Organisation: Organisation (User)</p>
+-->
+			<p>Items: {{OrderItems.length}}
+
+			<h4>Items</h4>
+			<div class='card card-body d-flex d-column gap-3'>
+			{{#each OrderItems}}
+				<div class="d-flex flex-row justify-content-between align-items-center">
+					<div>
+						<span> {{EntityType}}: {{Entity.custom.Name}} </span>
+						<span> {{{price Price}}} </span>
+					</div>
+					<div>
+						<a href='{{Entity.custom.Link}}' class="btn btn-default">View</a>
+					</div>
+				</div>
+			{{/each}}
+			</div>
+			{{/with}}
+		</div>
+	</div>
+</div>
+
+<div class="row text-center">
+	<div class="col-12 col-lg-6">
+		<div class="card card-body">
+			<h3>Payment</h3>
+
+			{{#eq order.Status "pending"}}
+			This order ({{{price order.Total}}}) has not been paid.
+
+			<form method="post" action="/order/{{order.id}}" class="card card-body mt-3">
```

Comment:
I think this needs to be slightly different

If I'm an Admin user, I should be able to set any status EXCEPT paid online - so that we can deal with cash/cheque/bank transfer payments

If I'm a non-admin user, I should either be able to pay via Square OR see details for cheque/bank transfer - and perhaps details of who to contact with queries etc

Reply:
Wait, I've just seen below!

I think it works better if the user just sends us a cheque, instead of having to select "Cheque" and then send us a cheque? So I suppose what I'm suggesting for Admins is merging this form and the awaiting confirmation form

Reply:
Current idea is:

Order is pending (not touched since creation)
Org sends money over bank, through cheque, hands cash, selects online
- If one of the manual (bank, cheque, cash) methods, the order goes into a "pending confirmation" status and square is used to create the payment. Now the treasurer (or admins for now) can verify this and confirm that it was paid. Once confirmed using the second form, the order will move to the "completed" status and square will be updated.
- If the online option is selected, the order goes into "processing" and square chugs away to get the online payment link. The user is redirected, they pay, we check with square and the order goes into "completed" without any manual intervention at all